runtime/centrifuge/src/lib.rs

```diff
@@ -348,45 +347,38 @@ pub enum ProxyType {
 	Any,
 	NonTransfer,
 	Governance,
-	// Staking,
-	// Vesting,
+	_Staking, // Deprecated ProxyType, that we are keeping due to the migration
+	NonProxy,
```

Comment:
```suggestion
	_Staking, 
	// Deprecated ProxyType, that we are keeping due to the migration
	NonProxy,
```

Did you mean like this?

Reply:
No, it refers to staking. Do you prefer it above?

Reply:
yeah, I find it best to have the comment above the thing it targets for consistency and clarity

Reply:
Alright, will change.